packages/util-utf8/package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "@aws-sdk/util-utf8",
+  "version": "3.0.0",
+  "description": "A UTF-8 string <-> UInt8Array converter",
+  "main": "./dist-cjs/index.js",
+  "module": "./dist-es/index.js",
+  "scripts": {
+    "build": "concurrently 'yarn:build:cjs' 'yarn:build:es' 'yarn:build:types'",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
+    "build:es": "tsc -p tsconfig.es.json",
+    "build:include:deps": "lerna run --scope $npm_package_name --include-dependencies build",
+    "build:types": "tsc -p tsconfig.types.json",
+    "build:types:downlevel": "downlevel-dts dist-types dist-types/ts3.4",
+    "clean": "rimraf ./dist-* && rimraf *.tsbuildinfo",
+    "test": "jest"
+  },
+  "author": {
+    "name": "AWS SDK for JavaScript Team",
+    "url": "https://aws.amazon.com/javascript/"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@aws-sdk/util-buffer-from": "*",
+    "tslib": "^2.3.1"
+  },
+  "devDependencies": {
+    "@tsconfig/recommended": "1.0.1",
+    "concurrently": "7.0.0",
+    "downlevel-dts": "0.10.1",
+    "rimraf": "3.0.2",
+    "typedoc": "0.19.2",
+    "typescript": "~4.6.2"
+  },
+  "types": "./dist-types/index.d.ts",
+  "engines": {
+    "node": ">=14.0.0"
+  },
+  "typesVersions": {
+    "<4.0": {
+      "dist-types/*": [
+        "dist-types/ts3.4/*"
+      ]
+    }
+  },
+  "files": [
+    "dist-*"
+  ],
```

Add configurations for browser and react-native
```suggestion
  ],
  "browser": {
    "./dist-es/fromUtf8": "./dist-es/fromUtf8.browser",
    "./dist-es/toUtf8": "./dist-es/toUtf8.browser"
  },
  "react-native": {
    "./dist-es/fromUtf8": "./dist-es/fromUtf8.browser",
    "./dist-es/toUtf8": "./dist-es/toUtf8.browser"
  },
```